src/Donors/resources/components/DonorsListTable.tsx

```diff
@@ -117,7 +117,7 @@ export default function DonorsListTable() {
             listTableBlankSlate={ListTableBlankSlate}
             productRecommendation={recommendation}
         >
-            <button className={styles.addFormButton} onClick={showLegacyDonors}>
+            <button className={`button button-tertiary ${styles.secondaryActionButton}`} onClick={showLegacyDonors}>
```

Comment:
Are we keeping the legacy screens?

Reply:
Yes, they will still exist until we have the new screens 100% compatible.